openssl-dynamic/pom.xml

```diff
@@ -107,6 +107,10 @@
           <family>windows</family>
         </os>
       </activation>
+      <properties>
+        <!-- Skip tests on windows as its not easy to load the openssl library we link against in the tests -->
+        <skipTests>true</skipTests>
```

Comment:
Can be removed once https://github.com/netty/netty-tcnative/pull/261 is merged :)